cosmos/models/Task.py

```diff
@@ -490,3 +491,10 @@ def __init__(self, **kwargs):
     @reconstructor
     def init_on_load(self):
         self.job_class = None
+
+    @property
+    def environment_variables_pretty(self):
```

Comment:
is this being used anywhere?

Reply:
actually it's not...I just added this method to duplicate `params_pretty` functionality, idk if you need it or not. It can be removed if not